mapbook-iOS/Info.plist

```diff
@@ -55,7 +55,7 @@
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
 	<key>NSPhotoLibraryUsageDescription</key>
-	<string>ArcGIS SDK requires access to the device&apos;s photo library.</string>
+	<string>ArcGIS SDK requires access to the device's photo library.</string>
```

Comment:
I know you said to ignore the changes in this file, but is this decoding expected?

Reply:
These sorts of changes are performed opportunistically by Xcode as the software evolves. It appears as if Apple has decided to move away from the escaped apostrophe in favor of using the UTF-8 apostrophe character instead. In fact, Xcode decided to sort our custom `AGSConfiguration` dictionary as well. These sorts of automatic changes happen all the time, especially with new releases of Xcode. There's nothing to worry about here.